src/Microsoft.DiaSymReader/SymUnmanagedFactory.cs

```diff
@@ -11,12 +11,12 @@ namespace Microsoft.DiaSymReader
 {
     internal static class SymUnmanagedFactory
     {
-        private const string AlternativeLoadPathEnvironmentVariableName = "MICROSOFT_DIASYMREADER_NATIVE_ALT_LOAD_PATH";
-        private const string AlternativeLoadPathOnlyEnvironmentVariableName = "MICROSOFT_DIASYMREADER_NATIVE_USE_ALT_LOAD_PATH_ONLY";
```

Comment:
>AlternativeLoadPathOnlyEnvironmentVariableName [](http://example.com/codeflow?start=29&length=46)

Removing this variable removes functionality. This was use to force loading from alternative path and skip standard load. This env var is documented here: [https://github.com/dotnet/symreader/blob/aec9e21f1cbc168511b96aee2dd73e8eeed5bc74/src/Microsoft.DiaSymReader/Reader/SymUnmanagedReaderCreationOptions.cs#L26](https://github.com/dotnet/symreader/blob/aec9e21f1cbc168511b96aee2dd73e8eeed5bc74/src/Microsoft.DiaSymReader/Reader/SymUnmanagedReaderCreationOptions.cs#L26)

I think we should preserve that behavior.

Reply:
Gotcha. Will add that back.

Reply:
Fixed that up.